### PR TITLE
Binder

### PR DESCRIPTION
**Pull request by sarimash**
### All Submissions

* [x] Have you followed the guidelines in our Contributing document?
* [x] Can you post a screenshot of your changes (if applicable)?
image

### New Feature Submissions

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally prior to submission?

### Changes to Core Features

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
binder

### CODE REVIEW

File: server/src/models/world/ai/behaviors/binder.ts [hunk 1]
```diff
@@ -15,8 +15,51 @@ export class BinderBehavior implements IAIBehavior {
...
+        // Bind the item
+        const item = player.items.equipment[ItemSlot.RightHand];
+        if (!item) return 'You must be holding an item in your right hand!';
+        if (item.mods.owner === player.username) return 'You already own that item!';
+        if (item.mods.owner && item.mods.owner !== player.username) return 'That item belongs to someone else!';
+
+        // Check for unbindable item classes
+        const itemClass = game.itemHelper.getItemProperty(item, 'itemClass');
+
+        if (itemClass === 'Corpse') return 'That is disrespectful.';
```
Comment (seiyria) [inline server/src/models/world/ai/behaviors/binder.ts:54]: use ItemClass.Corpse

File: server/src/models/world/ai/behaviors/binder.ts [hunk 2]
```diff
@@ -15,8 +15,51 @@ export class BinderBehavior implements IAIBehavior {
...
+        if (itemClass === 'Coin') return 'I can\'t engrave onto something so small.';
```
Comment (seiyria) [inline server/src/models/world/ai/behaviors/binder.ts:55]: use ItemClass.Coin

File: server/src/models/world/ai/behaviors/binder.ts (hunk 1 above)
Comment (sarimash) [reply server/src/models/world/ai/behaviors/binder.ts]: Thanks, done

File: server/src/models/world/ai/behaviors/binder.ts (hunk 2 above)
Comment (sarimash) [reply server/src/models/world/ai/behaviors/binder.ts]: Thanks, done

**Pull request merged**